cpp/src/Slice/Util.h

```diff
@@ -69,8 +69,11 @@ ciequals(const std::string& a, const std::string& b);
 std::string
 prependA(const std::string& s);
 
-TypePtr
-unwrapIfOptional(const TypePtr&);
+TypePtr unwrapIfOptional(const TypePtr& type);
+TypePtr unwrapIfAlias(const TypePtr& type);
+TypePtr unwrapType(const TypePtr& type);
```

Comment:
Same for unwrapType.

Reply:
I removed `unwrapType`, it was only used in one place, and only called the other two unwrap functions under the hood anyways.